moveit_ros/moveit_servo/src/servo_parameters.cpp

```diff
@@ -77,175 +83,363 @@ ServoParameters::setParametersCallback(std::vector<rclcpp::Parameter> parameters
   return result;
 }
 
-ServoParameters::SharedConstPtr ServoParameters::makeServoParameters(const rclcpp::Node::SharedPtr& node,
-                                                                     const rclcpp::Logger& logger,
-                                                                     std::string ns, /* = "moveit_servo"*/
-                                                                     bool dynamic_parameters /* = true */)
+void ServoParameters::declare(const std::string& ns,
+                              const rclcpp::node_interfaces::NodeParametersInterface::SharedPtr& node_parameters)
 {
-  auto parameters = std::shared_ptr<ServoParameters>(new ServoParameters(logger, ns));
+  using rclcpp::ParameterValue;
+  using rclcpp::ParameterType::PARAMETER_BOOL;
+  using rclcpp::ParameterType::PARAMETER_DOUBLE;
+  using rclcpp::ParameterType::PARAMETER_INTEGER;
+  using rclcpp::ParameterType::PARAMETER_STRING;
+  auto parameters = ServoParameters{};
 
-  // Get the parameters (organized same order as YAML file)
-  declareOrGetParam<bool>(parameters->use_gazebo, ns + ".use_gazebo", node, logger);
-  declareOrGetParam<std::string>(parameters->status_topic, ns + ".status_topic", node, logger);
+  // ROS Parameters
+  node_parameters->declare_parameter(
+      ns + ".use_gazebo", ParameterValue(parameters.use_gazebo),
```

Comment:
I'm sorry to be nit-picky about this but we should use brace initialization by default.

```suggestion
      ns + ".use_gazebo", ParameterValue{parameters.use_gazebo},
```